src/Xive/Hive/FileHive.cs

```diff
@@ -64,10 +64,7 @@ public FileHive(string scope, string root, Func<IHoneyComb, IHoneyComb> combWrap
             this.scope = scope;
             this.root = new Normalized(root);
             this.wrap = combWrap;
-            this.root = new Solid<string>(() =>
-            {
-                return new Normalized(root).AsString();
```

Comment:
You removed the stickyness of the value by removing Solid. Please restore the caching.